src/Tizen.Multimedia/Common.Internal/ValidationUtil.cs

```diff
@@ -44,5 +46,26 @@ internal static void ValidateFeatureSupported(string featureKey)
                 throw new NotSupportedException($"The feature({featureKey}) is not supported.");
             }
         }
+
+        internal static void ValidateIsNullOrEmpty(string value, string paramName)
```

Comment:
I checked, the caller of this method call it like..
ValidateIsNullOrEmpty(somestring, nameof(somestring));

I think, if the second parameter is used for just getting the name out of the first value, we can remove the second parameter, and get the name inside of this function.

Reply:
done